src/cpp11.cpp

```diff
@@ -197,34 +198,6 @@ extern "C" SEXP _readr_write_file_raw_(SEXP x, SEXP connection) {
 }
 
 extern "C" {
-/* .Call calls */
```

Comment:
These changes in cpp11.cpp are just due to using a newer version of cpp11 since the last time this was run, they can be safely ignored.